### DISCUSSION TITLE
error in linux

### DISCUSSION THREAD

**Discussion by hybridvamp** [BLUE-DEVIL1134/UltroidCli#18] [Q&A] [answered]
it showing error in linux like: "Ultroid: command not found"

screenshot [here](https://filetolinkbot0.herokuapp.com/91019)

**Comment by iAkashPattnaik** [comment, marked as answer]
> it showing error in linux like: "Ultroid: command not found"
> 
> screenshot -
![image](https://user-images.githubusercontent.com/55914808/147803721-d863c74e-71c9-436a-94b3-7730aedf43f2.png)

Okay, so read the whole Answer to understand completely !

## How to Install ?
```bash tab="terminal"
curl -L -o ultroid https://github.com/BLUE-DEVIL1134/UltroidCli/releases/download/1.0.6/ultroid-linux
```
 - This will download the **UltroidCli** to the path `./ultroid`
 - Replace `1.0.6` with the latest version available !

## How to Run ?
```bash tab="terminal"
./ultroid
```
 - This will show you the help menu !
 - Further, you can look at the [Docs](https://blue-devil1134.github.io/UltroidCli/) for more information !


⚠️ Star ⭐ The repo if you like it !
